backend/corpora/lambdas/api/v1/curation/collections/collection_id/actions.py

```diff
@@ -60,7 +61,14 @@ def patch(collection_id: str, body: dict, token_info: dict) -> Response:
             raise InvalidParametersHTTPException(detail="If provided, the 'links' array may not be empty")
         keep_links = False  # links have been provided; replace all old links
 
-    collection, errors = get_collection_and_verify_body(db_session, collection_id, body, token_info)
+    try:
+        collection, errors = get_collection_and_verify_body(db_session, collection_id, body, token_info)
+    except ForbiddenHTTPException as err:
+        if Collection.get_collection(
+            db_session, collection_id, CollectionVisibility.PUBLIC, owner=owner_or_allowed(token_info)
```

Comment:
A 405 would be more appropriate. It is what we use in other areas where a user tries to modify a public collection.

Reply:
Ah, that's a good point.

Reply:
fixed